docs/quickstarts/outlook-quickstart.md

```diff
@@ -91,12 +91,12 @@ The add-in project that you've created with the Yeoman generator contains sample
 ### Try it out
 
 > [!NOTE]
-> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
+> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides. You may also have to run your command prompt or terminal as an adminstrator for the changes to be made.
```

Comment:
I am not familiar with any administrator requirement.  Where is this coming from? Is there a particular issue that was raised around this?

Reply:
Yea so there were a couple errors where people were asked to rerun the npm run start:web command as administrator so that it could enable the loopback exemption. I ran through the flow this morning and hit the same thing. I think it's only required the first time they enable the exemption. After that, it's unneeded.

Reply:
Should I make the sentence more explicit about the loopback exemption?

Reply:
Oh, I see.  I think adding loopback exemption will only confused users, so let's just keep it as you have now

Reply:
Ok sounds good